src/TropicalGeometry/variety_affine_linear.jl

```diff
@@ -0,0 +1,22 @@
+################################################################################
+#
+#  Tropicalization of affine linear ideals
+#
+#  WARNING: assumes without test that `gens(I)` has degree one
+#
+################################################################################
+
+function tropical_variety_affine_linear(I::MPolyIdeal,nu::TropicalSemiringMap; weighted_polyhedral_complex_only::Bool=false)
+    # compute a reduced GB to check whether I is homogeneous
+    G = groebner_basis(I,complete_reduction=true)
+
+    if all(Oscar._is_homogeneous.(G))
```

Comment:
Also in other places :-)
```suggestion
    if all(Oscar._is_homogeneous,G)
```